examples/computer_vision/deformabledetr_coco_pytorch/model_def.py

```diff
@@ -0,0 +1,265 @@
+"""
+"""
+from collections import defaultdict
+from typing import Any, Dict, Sequence, Union
+from attrdict import AttrDict
+import numpy as np
+import sys
+import os
+import time
+
+sys.path.append("./ddetr")
```

Comment:
Do we need this line here? Can we clean up the directory structure somehow?

Reply:
unfortunately we need this since the original repo is not packaged up so we can't import from it without adding it to the python path.